my-site.rb

```diff
@@ -1,8 +1,20 @@
 require "sinatra"
 
 class MySite < Sinatra::Base
+  get "/about.html" do
+    erb :about
+  end
 
-  get "/" do
```

Comment:
It is still a "best practice" to have  "root route" that someone can go to directly when accessing your website